### ISSUE TITLE
Micromamba unable to install from url

### ISSUE THREAD

**Issue by urucoder** [mamba-org/mamba#2203]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### Search tried in issue tracker

micromamba install url

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not reproducible with Conda

### Describe your issue

I'm unable to install a package using the URL, the same URL works with Conda

**Not working:**

> micromamba install https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/linux-64/faiss-gpu-1.7.3-py3.9_h28a55e0_0_cuda11.3.tar.bz2

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/

https://anaconda.org/pytorch/faiss-gpu/1.7.3/dow..   3.3kB @   4.8kB/s 404 failed  0.7s
critical libmamba Multiple errors occured:
    Multi-download failed. Reason: Transfer finalized, status: 404 [https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch/repodata.json] 3283 bytes
    Subdir https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch not loaded!

**Working:**

> conda install https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/linux-64/faiss-gpu-1.7.3-py3.9_h28a55e0_0_cuda11.3.tar.bz2

Downloading and Extracting Packages
faiss-gpu-1.7.3 | ############################################################################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done

### mamba info / micromamba info

```shell
__
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/


            environment : test (active)
           env location : /opt/micromamba/envs/test
      user config files : /root/.mambarc
 populated config files : 
       libmamba version : 1.1.0
     micromamba version : 1.1.0
           curl version : libcurl/7.86.0 OpenSSL/1.1.1s zlib/1.2.13 libssh2/1.10.0 nghttp2/1.47.0
     libarchive version : libarchive 3.6.1 zlib/1.2.13 bz2lib/1.0.8 libzstd/1.5.2
       virtual packages : __unix=0=0
                          __linux=5.15.0=0
                          __glibc=2.35=0
                          __archspec=1=x86_64
               channels : 
       base environment : /opt/micromamba
               platform : linux-64
```

### Logs

```shell
__  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/

info     libmamba Parsing MatchSpec https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/linux-64/faiss-gpu-1.7.3-py3.9_h28a55e0_0_cuda11.3.tar.bz2
info     libmamba Searching index cache file for repo 'https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/linux-64/repodata.json'
info     libmamba No valid cache found
info     libmamba Using OpenSSL backend
info     libmamba Searching index cache file for repo 'https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch/repodata.json'
info     libmamba No valid cache found
info     libmamba Starting to download targets
[+] 0.4s
https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/linux-64 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━╸━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s  0.4s
https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch   ━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s  0.4sinfo     libmamba Transfer done for 'https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/linux-64'
https://anaconda.org/pytorch/faiss-gpu/1.7.3/dow..   3.3kB @   7.0kB/s 404 failed  0.5s
[+] 0.5s
https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch ━━━━━╸━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s  0.5sinfo     libmamba Transfer done for 'https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch'
info     libmamba Transfer finalized, status: 404 [https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch/repodata.json] 3284 bytes
https://anaconda.org/pytorch/faiss-gpu/1.7.3/dow..   3.3kB @   5.7kB/s 404 failed  0.6s
info     libmamba Freeing pool.
critical libmamba Multiple errors occured:
    Multi-download failed. Reason: Transfer finalized, status: 404 [https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch/repodata.json] 3284 bytes
    Subdir https://anaconda.org/pytorch/faiss-gpu/1.7.3/download/noarch not loaded!
```

### ~/.condarc

```yaml
cat: /root/.condarc: No such file or directory
```

**Comment by jonashaag** [comment]
Try downloading via browser first

**Comment by urucoder** [comment]
The link works, it even works via Conda

**Comment by jonashaag** [comment]
No I meant installing from disk not from URL

**Comment by aoot** [comment]
I'm also seeing the error message "no valid cache found" when trying to install a package. However, I was installing `xtensor` from conda-forge and I do have conda installed in my environment.

Thus, I am wondering if the cache issue is due to both conda and micromamba present on my system?

**Comment by AntoinePrv** [comment]
Thanks for reporting, this should be now solved in `main` and available in the next 2.0 release.
